lib/subcommands/ui.js

```diff
@@ -36,4 +36,91 @@ commands.setAppearance = async function setAppearance (appearance) {
   });
 };
 
+/**
+ * Retrieves the current increase contrast configuration value from the given simulator.
+ * The value could be:
+ *   - enabled: Increase Contrast is enabled.
```

Comment:
I assume it would be more useful to add the list of possible values to the param documentation. Also applies to below methods